aiven/client/client.py

```diff
@@ -2140,3 +2140,20 @@ def detach_team_from_project(self, account_id: str, team_id: str, project: str)
             self.delete,
             self.build_path("account", account_id, "team", team_id, "project", project),
         )
+
+    def create_oauth2_client(self, account_id: str, name: str, description: Optional[str] = None) -> Mapping:
+        return self.verify(
+            self.post,
+            self.build_path("account", account_id, "oauth_client"),
+            body={"name": name, "description": description},
+        )
+
+    def add_oauth2_client_redirect(self, account_id: str, client_id: str, uri: str) -> Mapping:
+        return self.verify(
+            self.post,
+            self.build_path("account", account_id, "oauth_client", client_id, "redirect"),
+            body={"redirect_uri": uri},
+        )
+
+    def add_oauth2_client_secret(self, account_id: str, client_id: str) -> Mapping:
```

Comment:
shouldnt this be `get_oauth2....` ?

Reply:
Maybe create? It does post request and issues new one.